client/app/instructor/verifystudents/verifystudents.html

```diff
@@ -0,0 +1,34 @@
+<navbar></navbar>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-3 sidebar">
+          <sidebar></sidebar>
+        </div>
+        <div class="col-sm-9 main">
+          <h1 class="page-header">Verify Students</h1>
+          <div class = "row">
+            <div class="col-sm-12">
+              <div class="well" ng-repeat="section in sections">
+                <div class = "section-number">
+                  <h3 class = "section-number" >Sections: <div class = "section-number" ng-repeat="num in section.sectionNumbers"> {{num}} 	&nbsp;</div> </h3>
+                </div>
+                <div ng-if="section.pendingStudents.length == 0">
+                  No Pending Students
+                </div>
+                <div ng-if="section.pendingStudents.length != 0">
+
+                  <ul id="check-list-box" class="list-group checked-list-box">
+                     <li class="list-group-item student-name" ng-repeat="pendingStudent in section.pendingStudents">
+                        <h5> {{pendingStudent.firstName}} {{pendingStudent.lastName}} </h5>
```

Comment:
oi this is okay but i wish we could just say `pendingStudent.name`- that's a fix for the backend